database/factories/CompanyFactory.php

```diff
@@ -1,17 +1,17 @@
-<?php
-
-/** @var \Illuminate\Database\Eloquent\Factory $factory */
-
-use App\Company;
-use Faker\Generator as Faker;
-
-$factory->define(Company::class, function (Faker $faker) {
-    return [
-        //
-        'name' => $faker->name,
-        'shortname' => $faker->word(10),
-        'industry' => $faker->company,
-        'ceo' => $faker->name(2),
-        'twitter' => $faker->unique()->firstName,
-    ];
-});
+<?php
+
+/** @var \Illuminate\Database\Eloquent\Factory $factory */
+
+use App\Company;
+use Faker\Generator as Faker;
+
+$factory->define(Company::class, function (Faker $faker) {
+    return [
+        //
+        'snumber' => $faker->snumber,
+        'service' => $faker->service,

```

Comment:
why are you changing the table name?

Reply:
I changed it with respect to the valid information I could get online

Reply:
I changed it to suit what we have best

Reply:
What changes do you need me to make before you merge this?